web/src/components/SpeakerCard.vue

```diff
@@ -72,6 +72,9 @@ export default {
     },
     name() {
       try {
+        if (this.$i18n.locale === 'ja') {
+          return this.speaker.get('name_ja') || '';
```

Comment:
If there's no Japanese name, shouldn't it fallback to English instead of empty string?

Reply:
True... let me do a little more investigation of what airtable returns on a blank record, if that kind of error can happen.

Reply:
From the docs: "Any "empty" fields (e.g. "", [], or false) in the record will not be returned."